inc/html.class.php

```diff
@@ -5753,6 +5770,15 @@ static function textarea($options = []) {
                         });
                      ");
       }
+      if (!$p['enable_fileupload'] && $p['enable_richtext']) {
```

Comment:
What are those new options? I'm not sur I understand the condition here.

Reply:
The code in the if() statement generates an area where is displayed the file uploads. Example when a user crates a ticket

![image](https://user-images.githubusercontent.com/14139801/75783664-36c03b00-5d61-11ea-93f1-488c6c1633bc.png)

Instead of displaying the uploads of the textarea (aka screenshots)  in the file input, those uplaods are displayed just under the textarea. 

For backward compatibility, just in case, I avoid to generate the file uploads area if a textarea is explicitly linked to a file input via the options. This makes sense : when enable_fileuploads is false, only screenshots are allowed. If true a file input will be generated by the call to textarea()

Reply:
additional note : the latest case exists when creating a ticket  task

Reply:
Still not sure I understand, but `enable_richtext` should probably not be used.

Reply:
Look the occurences of calls to Html::uploadedFiles()

There is an occurence in Html::file() itself called by Html::textare() when enable_fileuploads is true.

When the Html::textarea is called with enable_fileuploads == false then we may still need to support uploads of screenshots embedded in the text. 

This is needed in Formcreator to allow rich text in a texstarea without making visible a file upload area (drag  /drop area + browse button).

Moreover it may be an interesting target to remove all raw textarea in GLPI code (see ```git grep "<textarea"```) . I did the pr with this target in mind. At least it will help me to reach this goal in plugins.